core/src/main/java/org/mule/runtime/core/api/message/ExceptionPayload.java

```diff
@@ -19,8 +19,6 @@
 @Deprecated
 public interface ExceptionPayload extends Serializable {
 
-  int getCode();
```

Comment:
Check if META-INF/org/mule/runtime/core/config/mule-exception-codes.properties and META-INF/org/mule/runtime/core/config/mule-exception-config.properties can be deleted too